packager/react-native-xcode.sh

```diff
@@ -46,7 +46,14 @@ fi
 # npm global install path may be a non-standard location
 PATH="$(npm prefix -g)/bin:$PATH"
 
-react-native bundle \
+# check for NPM-installed react-native-cli
+if [[ -d ./node_modules/react-native-cli ]] && [ -z "$REACT_NATIVE_PATH" ]; then
```

Comment:
The generated Xcode project's build step also relies on cwd, so (I think) it's a consistent assumption:

<img width="674" alt="screen shot 2016-01-19 at 7 53 00 am" src="https://cloud.githubusercontent.com/assets/153704/12423712/ae733964-be81-11e5-891b-cfd3798447a2.png">